### ISSUE TITLE
Minetweaker Recipes won't work with the crafting table.  

### ISSUE THREAD

**Issue by delet3r** [Elecs-Mods/CraftingTable-IV#34]
It shows the sub items, but won't craft anything.  For example I made it so abyssal cobble is crafted iwth 9 cobble (like compressed cobble) and then had stone picks crafted with abyssal cobble.  The table will show abyssal cobble on the grid, and the stone pick, when i have cobble and wood in my inventory, but I can't craft it.  When I click the stone pick, nothing happens.

So it appears it is accessing the recipes I added in minetweaker, but it won't actually make the item for some reason.

**Comment by draknyte1** [comment]
If you don't have the correct wood items, I find it won't make them.
If you need sticks, have sticks not planks or logs. (Minetweaker recipes work fine for me, I've made over 1000 items in a modpack with 150 minetweaked mods & Gregtech 5)

**Comment by delet3r** [comment]
Thats odd.  In my example about abyssal stone, I tried making just the abyssal from cobble.  The abyssal cobble showed in the tables gui, but when I click it, still nothing happens.  Im pretty sure i tried making the abyssal by hand too and then having it craft a pickaxe, and it wouldn't work.  

Ok ... wait I just realized I misinterpreted your comment, you mean only the WOOD items need to be correct?  Is that for all items or just minetweaked recipes?  Actually has to be minetweaked, I made an iron axe yesterday just fine.  

Ill try it and make the sticks myself, then see if it makes the pickaxe out of abyssal cobble.
